galcheat/data/LSST.yaml

```diff
@@ -59,23 +59,23 @@ references:
     link: "https://www.lsst.org/about/tel-site/optical_design"
     comment: ""
   obscuration:
-    link: "https://arxiv.org/abs/0805.2366"
-    comment: "See https://github.com/aboucaud/galcheat/issues/38 for discussions"
+    link: "https://arxiv.org/abs/0805.2366v4.pdf"
+    comment: "The paper mentions an effective diameter of ~6.5m. See https://github.com/aboucaud/galcheat/issues/38 for discussions on how to calculate obscuration from effective diameter."
   zeropoint_airmass:
     link: "https://speclite.readthedocs.io/en/latest/filters.html#lsst-filters"
     comment: ""
   sky_brightness:
-    link: "https://www.lsst.org/about/camera/features"
-    comment: ""
+    link: "https://arxiv.org/pdf/0805.2366v4.pdf"
+    comment: "See Table 2, first row"
   exposure_time:
-    link: "https://www.lsst.org/about/camera/features"
-    comment: "See Technical details"
+    link: "https://arxiv.org/pdf/0805.2366v4.pdf"
+    comment: "See Table 1 'Mean number of visits' and in the paper it is mentioned that visit exposure time 30s. So for example, for u band, 56 * 30 = 1680."
```

Comment:
Here also, is there a reason to prefer the arXiv ref over the project official website ?
I have no opinion. 

@mpaillassa ?

Reply:
Hm yes I tend to favor arXiv references but this is subjective and we may discuss about this.
But in this case `exposure_time` is in the arXiv reference but I can't find it in the [link](https://www.lsst.org/about/camera/features).

Reply:
In `Camera Data Rates` it says _15 sec integration_. But for this one I guess it's best if we refer directly to a paper stating the exposure time.

Reply:
We can discuss in the meeting tomorrow, but I think I prefer referring the paper directly as well.